app/src/main/java/ch/epfl/sweng/erpa/model/Game.java

```diff
@@ -11,31 +16,35 @@
 import lombok.NoArgsConstructor;
 import lombok.NonNull;
 
+@Entity
 @Data
 @Builder(toBuilder = true)
 @NoArgsConstructor
 @AllArgsConstructor
 public class Game implements UuidObject {
-    @NonNull private String uuid;
+    @android.support.annotation.NonNull @PrimaryKey @NonNull private String uuid = "";
+    @ForeignKey(entity = UserProfile.class, parentColumns = "uuid", childColumns = "gm_user_uuid")
     @NonNull private String gmUserUuid;
-    @NonNull private String title;
-    @NonNull private Integer minPlayers;
-    @NonNull private Integer maxPlayers;
-    @NonNull private Difficulty difficulty;
-    @NonNull private String universe;
-    @NonNull private Boolean isCampaign;
-    @NonNull private Optional<Integer> numberOfSessions;
-    @NonNull private Optional<Integer> sessionLengthInMinutes;
-    @NonNull private String description;
-    @NonNull private Double locationLat;
-    @NonNull private Double locationLon;
-    @NonNull private GameStatus gameStatus;
+
+    @ColumnInfo @NonNull private String title;
+    @ColumnInfo(name = "min_players") @NonNull private Integer minPlayers;
+    @ColumnInfo(name = "max_players") @NonNull private Integer maxPlayers;
+    @ColumnInfo @NonNull private Difficulty difficulty;
+    @ColumnInfo @NonNull private String universe;
+    @ColumnInfo @NonNull private Boolean isCampaign;
+    @ColumnInfo @NonNull private Optional<Integer> numberOfSessions;
```

Comment:
Did you forget to specify the column name? or the name attribute in lines 30 and 31 is superfluos?